packages/mask/src/plugins/EVM/services/Asset/getHistory.ts

```diff
@@ -0,0 +1,24 @@
+import { ChainId, NonFungibleAssetProvider } from '@masknet/web3-shared-evm'
+
+import { unreachable } from '@dimensiondev/kit'
+import { NFTScanApi, OpenSeaApi, RaribleApi } from '@masknet/web3-providers'
+
+export async function getHistory(
+    address: string,
+    tokenId: string,
+    chainId: ChainId,
+    provider: NonFungibleAssetProvider,
+    page: number,
+    size: number,
+) {
```

Comment:
plz extract the function parameters to interface definition

```suggestion
interface OpenSeaHistoryOptions {
    provider: NonFungibleAssetProvider.OPENSEA
    // ... other params ...
}

// etc, more options interface definition

type HistoryOptions = OpenSeaHistoryOptions | NFTScanHistoryOptions | RaribleHistoryOptions

export async function getHistory(options: HistoryOptions) {
```